xml/System.Windows.Forms.Design.Behavior/BehaviorServiceAdornerCollection.xml

```diff
@@ -271,6 +271,71 @@
         <param name="index">The index in <paramref name="array" /> where copying begins.</param>
         <summary>Copies the <see cref="T:System.Windows.Forms.Design.Behavior.BehaviorServiceAdornerCollection" /> values to a one-dimensional <see cref="T:System.Array" /> at the specified index.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="array" /> is multidimensional.
+             -or-
+             The number of elements in the
+             <see cref="T:System.Windows.Forms.Design.Behavior.BehaviorServiceAdornerCollection" /> is greater than the
+             available space between <paramref name="index" /> and the end of <paramref name="array" />.
+            </exception>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="array" /> is <see langword="null" />. </exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="arrayIndex" /> is less than
+             <paramref name="array" />'s lowbound.
+            </exception>
+        <exception cref="T:System.ArgumentException">
```

Comment:
there are several duplicate exceptions here

Reply:
Indeed. Here's the xml-doc this was generated from 
https://github.com/dotnet/winforms/blob/7aeb736b075b102efb4d3fc4df6add660177d032/src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/AdornerCollection.cs#L128-L157

The only thing I can speculately think of - there is a bug in the tool that causes definition to be added multiple times...
![image](https://user-images.githubusercontent.com/4403806/62863628-04da7f00-bd12-11e9-86ab-8f25d99bef91.png)

I have thought it may be related to [this line](https://github.com/dotnet/winforms/blob/7aeb736b075b102efb4d3fc4df6add660177d032/src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/AdornerCollection.cs#L153), but having cleaned and regenerated binaries and re-run the tool the result was the same.

I haven't had a chance to look further at this stage.

Reply:
Is this caused by @carlossanlop's tool?

Reply:
Oh, it probably was caused by my tool. I thought I had the issue fixed. I will take a look again.
@RussKie I'll help add suggestions to your changes so you can remove the duplicates, if I still find some.

Reply:
Update: I removed them myself. Much easier than adding individual suggestions :). Please take a look.

Reply:
Created a new PR using our CI tool updates: https://github.com/dotnet/dotnet-api-docs/pull/2984